src/main/java/org/molgenis/vcf/report/model/Phenopacket.java

```diff
@@ -0,0 +1,19 @@
+package org.molgenis.vcf.report.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import lombok.NonNull;
+import lombok.Value;
+import lombok.experimental.NonFinal;
+
+@Value
+@NonFinal
+public class Phenopacket {
+  @JsonProperty("phenotypicFeaturesList")
```

Comment:
should be phenotypicFeatures instead of phenotypicFeaturesList (https://phenopackets-schema.readthedocs.io/en/latest/phenopacket.html)